test/index.js

```diff
@@ -451,4 +473,79 @@ describe('KeyringController', function () {
       );
     });
   });
+
+  describe('Manifest v3 changes', function () {
+    it('can still verify password', async function () {
+      // Log the user out
+      await keyringController.setLocked();
+
+      // Attempt to verify the password
+      const result = await keyringController.verifyPassword(password);
```

Comment:
(nit) verifyPassword throws on failure, so the assertion is not reached if the test won't pass.
using async assertions may produce more helpful output from Jest
```suggestion
     await expect(keyringController.verifyPassword(password)).resolves.toBeTruthy()
```